### DISCUSSION TITLE
Improve license normalization

### DISCUSSION THREAD

**Discussion by pbaumard** [aquasecurity/trivy#7130] [Ideas]
### Description

Some unmatched licenses with trivy 0.53 with a sample corresponding Maven package name:

- "MIT License": org.slf4j:slf4j-api
- "The MIT License": org.mockito:mockito-core
- "BSD License 3": org.hamcrest:hamcrest
- "Eclipse Public License 1.0": junit:junit
- "GPL2 w/ CPE": jakarta.annotation:jakarta.annotation-api
- "MIT-0": org.reactivestreams:reactive-streams

### Target

None

### Scanner

License

**Comment by pbaumard** [comment]
Best as an issue for the related PR #7131

**Comment by pbaumard** [comment]
Reopening.

https://aquasecurity.github.io/trivy/v0.53/community/contribute/pr/ is misleading with

> Every Pull Request should have an associated bug or feature issue

But actually as written in https://aquasecurity.github.io/trivy/v0.53/community/contribute/issue/

> Trivy uses [GitHub Discussion](https://aquasecurity.github.io/trivy/v0.53/community/contribute/discussion/) for bug reports, feature requests, and questions.

**Reply by knqyf263** [reply]
Thanks for your feedback.

>If maintainers decide to accept a new feature or confirm that it is a bug, they will close the discussion and create a [GitHub Issue](https://github.com/aquasecurity/trivy/issues) associated with that discussion.

As written here, an issue is created after discussion, then PR should be associated with the issue. This is because PRs sent before the Issue is generated (i.e. before the discussion takes place) may not be accepted.

As you pointed out, the document is probably misleading. We welcome document updates.

**Reply by pbaumard** [reply]
Fixed in https://github.com/aquasecurity/trivy/pull/7141

**Reply by knqyf263** [reply]
Thanks for your contribution.

**Reply by knqyf263** [reply]
Opened an issue for track
https://github.com/aquasecurity/trivy/issues/7189
